docs/doc/30-reference/20-functions/122-window-functions/aggregate-window-functions.md

```diff
@@ -56,9 +59,13 @@ SELECT date, AVG(amount) AS avg_amount_for_branch
 FROM BookSold
 GROUP BY date;
 
-June 21|544.0
-June 22|454.5
-June 23|643.0
++---------+-----------------------+
```

Comment:
The result format style is not same as others, here is a table :)

Reply:
> The result format style is not same as others, here is a table :)

Revert to previous format?

Reply:
Ok